website/components/remote-plugin-docs/utils/resolve-nav-data.js

```diff
@@ -132,7 +132,13 @@ async function mergeRemotePlugins(remotePluginsFile, navData, currentPath) {
 // which contains { filePath, fileString } data for the remote
 // plugin doc .mdx file
 async function resolvePluginEntryDocs(pluginConfigEntry, currentPath) {
-  const { title, path: slug, repo, version } = pluginConfigEntry
+  const {
+    title,
+    path: slug,
+    repo,
+    version,
+    sourceBranch = 'main',
```

Comment:
Apart from the formatting, the change here is `sourceBranch`, with the the `"main"` default.

Note that both plugins currently configured in remote repos use the default `main` value, so the plugin docs configuration file did not need to be modified.

Reply:
Nice!